### DISCUSSION TITLE
I can no longer assign domains to my projects

### DISCUSSION THREAD

**Discussion by romellogoodman** [vercel/community#6327] [Help] [answered]
### Summary

I received a notification that my account was updated to a new free plan and now I can no longer assign domains to my projects. 

### Steps to Reproduce

I dont know how you would recreate this on your end.

**Comment by swarnava** [comment]
Can you send a screenshot?

**Comment by amyegan** [comment, marked as answer]
Hey @romellogoodman. Do to solutions posted in https://github.com/vercel/community/discussions/6298 work for you?

**Comment by romellogoodman** [comment]
Yes that works, thank you!
